### ISSUE TITLE
p2ch11 我尝试运行training这个文件,并对LunaModel 略做修改,两个结果有些不同,我不知道哪一个结果更好?

### ISSUE THREAD

**Issue by smd-1886** [deep-learning-with-pytorch/dlwpt-code#97]
我想知道下面的两个结果哪一个更好一些?
下面的结果,是我对LunaModel进行修改后的.
# 2022-09-03 09:43:48,765 INFO     pid:12876 __main__:081:initModel Using CUDA; 1 devices.
# 2022-09-03 09:43:49,618 INFO     pid:12876 __main__:140:main Starting LunaTrainingApp, Namespace(batch_size=32, comment='dwlpt', epochs=1, num_workers=8, tb_prefix='p2ch11')
# 2022-09-03 09:38:40,854 INFO     pid:11603 dsets:191:__init__ <dsets.LunaDataset object at 0x7f6486b69340>: 51244 training samples
# 2022-09-03 09:38:40,856 INFO     pid:11603 dsets:191:__init__ <dsets.LunaDataset object at 0x7f63966d2fd0>: 5694 validation samples
# 2022-09-03 09:38:40,856 INFO     pid:11603 __main__:147:main Epoch 1 of 1, 1602/178 batches of size 32*1
# 2022-09-03 09:38:40,856 WARNING  pid:11603 util.util:219:enumerateWithEstimate E1 Training ----/1602, starting
# 2022-09-03 09:38:45,845 INFO     pid:11603 util.util:236:enumerateWithEstimate E1 Training   64/1602, done at 2022-09-03 09:40:06, 0:01:23
# 2022-09-03 09:38:55,958 INFO     pid:11603 util.util:236:enumerateWithEstimate E1 Training  256/1602, done at 2022-09-03 09:40:06, 0:01:23
# 2022-09-03 09:39:36,904 INFO     pid:11603 util.util:236:enumerateWithEstimate E1 Training 1024/1602, done at 2022-09-03 09:40:07, 0:01:24
# 2022-09-03 09:40:08,145 WARNING  pid:11603 util.util:249:enumerateWithEstimate E1 Training ----/1602, done at 2022-09-03 09:40:08
# 2022-09-03 09:40:08,282 INFO     pid:11603 __main__:261:logMetrics E1 LunaTrainingApp
# 2022-09-03 09:40:08,282 INFO     pid:11603 __main__:291:logMetrics E1 trn      nan loss,  36.1% correct,
# 2022-09-03 09:40:08,282 INFO     pid:11603 __main__:300:logMetrics E1 trn_neg  nan loss,  36.0% correct (18433 of 51135)
# 2022-09-03 09:40:08,282 INFO     pid:11603 __main__:311:logMetrics E1 trn_pos  nan loss,  66.1% correct (72 of 109)
# 2022-09-03 09:40:08,289 WARNING  pid:11603 util.util:219:enumerateWithEstimate E1 Validation  ----/178, starting
# 2022-09-03 09:40:09,335 INFO     pid:11603 util.util:236:enumerateWithEstimate E1 Validation    64/178, done at 2022-09-03 09:40:10, 0:00:02
# 2022-09-03 09:40:11,296 WARNING  pid:11603 util.util:249:enumerateWithEstimate E1 Validation  ----/178, done at 2022-09-03 09:40:11
# 2022-09-03 09:40:11,540 INFO     pid:11603 __main__:261:logMetrics E1 LunaTrainingApp
# 2022-09-03 09:40:11,541 INFO     pid:11603 __main__:291:logMetrics E1 val      nan loss,   0.2% correct,
# 2022-09-03 09:40:11,541 INFO     pid:11603 __main__:300:logMetrics E1 val_neg  nan loss,   0.0% correct (0 of 5681)
# 2022-09-03 09:40:11,541 INFO     pid:11603 __main__:311:logMetrics E1 val_pos  nan loss, 100.0% correct (13 of 13)
#
# Process finished with exit code 0

下面的结果是原LunaModel没有修改过的
# 2022-09-03 09:43:48,765 INFO     pid:12876 __main__:081:initModel Using CUDA; 1 devices.
# 2022-09-03 09:43:49,618 INFO     pid:12876 __main__:140:main Starting LunaTrainingApp, Namespace(batch_size=32, comment='dwlpt', epochs=1, num_workers=8, tb_prefix='p2ch11')
# 2022-09-03 09:43:50,876 INFO     pid:12876 dsets:191:__init__ <dsets.LunaDataset object at 0x7f4852361340>: 51244 training samples
# 2022-09-03 09:43:50,878 INFO     pid:12876 dsets:191:__init__ <dsets.LunaDataset object at 0x7f4761eccfd0>: 5694 validation samples
# 2022-09-03 09:43:50,878 INFO     pid:12876 __main__:147:main Epoch 1 of 1, 1602/178 batches of size 32*1
# 2022-09-03 09:43:50,879 WARNING  pid:12876 util.util:219:enumerateWithEstimate E1 Training ----/1602, starting
# 2022-09-03 09:43:55,816 INFO     pid:12876 util.util:236:enumerateWithEstimate E1 Training   64/1602, done at 2022-09-03 09:45:16, 0:01:24
# 2022-09-03 09:44:05,928 INFO     pid:12876 util.util:236:enumerateWithEstimate E1 Training  256/1602, done at 2022-09-03 09:45:16, 0:01:23
# 2022-09-03 09:44:46,939 INFO     pid:12876 util.util:236:enumerateWithEstimate E1 Training 1024/1602, done at 2022-09-03 09:45:17, 0:01:24
# 2022-09-03 09:45:18,085 WARNING  pid:12876 util.util:249:enumerateWithEstimate E1 Training ----/1602, done at 2022-09-03 09:45:18
# 2022-09-03 09:45:18,223 INFO     pid:12876 __main__:261:logMetrics E1 LunaTrainingApp
# 2022-09-03 09:45:18,224 INFO     pid:12876 __main__:291:logMetrics E1 trn      0.1799 loss,  99.7% correct,
# 2022-09-03 09:45:18,224 INFO     pid:12876 __main__:300:logMetrics E1 trn_neg  0.1162 loss,  99.9% correct (51085 of 51135)
# 2022-09-03 09:45:18,224 INFO     pid:12876 __main__:311:logMetrics E1 trn_pos  30.0633 loss,   0.0% correct (0 of 109)
# 2022-09-03 09:45:18,228 WARNING  pid:12876 util.util:219:enumerateWithEstimate E1 Validation  ----/178, starting
# 2022-09-03 09:45:19,233 INFO     pid:12876 util.util:236:enumerateWithEstimate E1 Validation    64/178, done at 2022-09-03 09:45:20, 0:00:02
# 2022-09-03 09:45:21,192 WARNING  pid:12876 util.util:249:enumerateWithEstimate E1 Validation  ----/178, done at 2022-09-03 09:45:21
# 2022-09-03 09:45:21,443 INFO     pid:12876 __main__:261:logMetrics E1 LunaTrainingApp
# 2022-09-03 09:45:21,443 INFO     pid:12876 __main__:291:logMetrics E1 val      0.0162 loss,  99.8% correct,
# 2022-09-03 09:45:21,443 INFO     pid:12876 __main__:300:logMetrics E1 val_neg  0.0020 loss, 100.0% correct (5681 of 5681)
# 2022-09-03 09:45:21,443 INFO     pid:12876 __main__:311:logMetrics E1 val_pos  6.2100 loss,   0.0% correct (0 of 13)
